be/src/olap/memtable.cpp

```diff
@@ -64,10 +64,10 @@ MemTable::MemTable(TabletSharedPtr tablet, Schema* schema, const TabletSchema* t
     _insert_mem_tracker_use_hook = std::make_unique<MemTracker>(
             fmt::format("MemTableHookInsert:TabletId={}", std::to_string(tablet_id())));
 #endif
-    _table_mem_pool = std::make_unique<MemPool>(_insert_mem_tracker.get());
```

Comment:
insert mem tracker is lost???

Reply:
Arena add `allocated_size() `, `MemTable::memory_usage()` contains `arena->allocated_size()`